R/compare_designs.R

```diff
@@ -1,8 +1,51 @@
-compare_partial <- function(FUN, DIFFFUN){
-  function(design1, design2, format = "ansi256", mode = "sidebyside", pager = "off", context = -1L, rmd = FALSE){
-    stopifnot(requireNamespace("diffobj"))
-    DIFFFUN <- get(DIFFFUN, getNamespace("diffobj"))
-    compare_design_internal(FUN, DIFFFUN, design1, design2, format = "ansi256", mode = "sidebyside", pager = "off", context = -1L, rmd = FALSE)
+compare_partial <- function(FUN, DIFFFUN, is_data = FALSE){
```

Comment:
I would just make two functions and call the appropriate one - see also https://martinfowler.com/bliki/FlagArgument.html

Reply:
fair, will do